client/state/signup/steps/reducer.js

```diff
@@ -14,6 +15,7 @@ import survey from './survey/reducer';
 export default combineReducers( {
 	designType,
 	siteTitle,
+	siteTopic,
```

Comment:
Should we add it to the bundle later? Assuming this is going out first?

Reply:
It should be safe, because nothing really uses this substate tree.

Reply:
It was mainly to avoid extra bytes to the bundle where don't yet need to.

Reply:
Ohhhhh. That makes sense. I'll remove it from here before deploying it.